deployment/kubernetes/aerie-ui-deployment.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: aerie-dev
+  name: aerie-ui
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: aerie-ui
+  template:
+    metadata:
+      labels:
+        app: aerie-ui
+    spec:
+      containers:
+        - env:
+            - name: ORIGIN
+              value: http://127.0.0.1
```

Comment:
How is `ORIGIN` set for the deployment?

Reply:
`ORIGIN` is currently set to `http://localhost` for the `docker-compose.yaml` based dev deployment so I imagine it could stay the same if/when we switch to the k8s based deployment. Those current env var values are the result of a bit of trial and error getting CORS to work.

Reply:
So the deployment docker-compose.yaml right now assumes you are running locally. If we were to use this to deploy to AWS how do we update these env vars? Is there a file on the host we can use to overwrite them?

Reply:
Would this also use [kustomize](https://kustomize.io/) to update env vars?

Reply:
Once an AWS EKS cluster is set up, it will provide authentication tokens that can be used with `kubectl` to control the cluster remotely, e.g. `kubectl apply -f deployment/kubernetes` would deploy Aerie to the cluster that `~/.kube/config` currently points to. To customize a deployment / environment variables, I see three main options:

- Users maintain their own manifests with hardcoded values (least attractive option)
- The environment variables could be patched onto our base manifests with `kustomize` and applied directly using `kubectl apply -k .`, i.e. users maintain their own `kustomize.yaml` file.
- The base manifests could be updated to to populate env vars them from a `Secret` or `ConfigMap`. End users could then maintain a `.env` file and create a `ConfigMap` with something like `kubectl -n aerie-dev create configmap dev-env --from-env-file=.env`. 

The last option is how I was managing secrets while testing these manifests; right now only "secrets" are managed with an `.env` file, e.g. `AERIE_PASSWORD` is stored in a secret that has to be manually created, and then `aerie-merlin-deployment` can [reference that secret](https://github.com/NASA-AMMOS/aerie/pull/606/files#diff-cb7b2a918b62b075a81c780d77733b7bfe0101ce1ca04ae16ad50a8c9a73ae91R45) to create the `MERLIN_DB_PASSWORD` env var. We could extend this same technique to all environment variables.

Reply:
Note that these `kubectl` commands could easily be applied in a `deploy` GitHub action, so this whole thing could be automated.